AuthVO.tex

```diff
@@ -883,10 +901,13 @@ \subsection{Mandatory authentication with certificates}
 is required to access the service.
 The \verb|Bearer| challenge (\rfc{6750}) means we can authenticate with
 a Bearer Token if we have or know how to get one, but we don't.
-We can however use the \verb|ivoa_x509| challenge
-(Section~\ref{sec:ivoa-x509});
-it has a \verb|standard_id| of \verb|BasicAA| (Section~\ref{sec:standard-id})
-so transmit user credentials using
+The unparameterised \verb|ivoa_x509| challenge means we can authenticate
+with a certificate if we have one, but we don't.
```

Comment:
I would probably state this as 
```
The unparameterised \verb|ivoa_x509| challenge means the client can, in principle, authenticate with a certificate from any valid CA and not just one issued by the endpoint in the parameterised challenge.
```
I think the rest correctly states that the parameterised challenge says how to obtain a client certificate *and* that the service accepts such (locally issued) certificates.